localization_folder: "/home/zjl/semantic_3d_avp_ws/src/semantic_3d_avp/data/zjg2_2023-3-18-2/parking/"
start_frame: 0
end_frame: 81